tox.ini

```diff
@@ -1,36 +1,28 @@
 [tox]
 envlist =
-    py27, py34,
-    docs, flake8
```

Comment:
It's great that you're supporting so many python versions! Write that into the ``README``